docs/src/app/components/pages/components/Stepper/VerticalNonLinearStepper.js

```diff
@@ -15,6 +15,7 @@ class VerticalNonLinear extends React.Component {
 
   state = {
     stepIndex: 0,
+    test: true,
```

Comment:
Do you want to keep this state in the PR? I feel like you have forgotten to remove it 😁 .